discord/flags.py

```diff
@@ -1338,6 +1338,14 @@ def group_dm_create(self):
         """:class:`bool`: Returns ``True`` if the application can create group DMs."""
         return 1 << 5
 
+    @flag_value
+    def application_auto_moderation_rule_create_badge(self):
+        """:class:`bool`: Returns ``True`` if the application uses the Auto Moderation API
```

Comment:
you got rid of the period again
```suggestion
        """:class:`bool`: Returns ``True`` if the application uses the Auto Moderation API.
```